package.json

```diff
@@ -57,8 +57,8 @@
     "protractor": "~5.1.2",
     "request-promise": "^4.2.1",
     "ts-node": "~3.0.4",
-    "tslint": "~5.3.2",
+    "tslint": "^5.8.0",
     "tslint-eslint-rules": "^4.1.1",
-    "typescript": "~2.3.3"
+    "typescript": "^2.6.1"
```

Comment:
Please don't change the version of TypeScript in the `package.json`.  This needs to match the recommended version for Angular.

Reply:
Sorry, that was due to NPM install. I have replaced the package.json with package.json from the master, for the sake of consistency.